glide.lock

```diff
@@ -389,7 +391,7 @@ imports:
   - go/types/typeutil
   - imports
 - name: google.golang.org/api
-  version: 16ab375f94503bfa0d19db78e96bffbe1a34354f
+  version: 48e49d1645e228d1c50c3d54fb476b2224477303
```

Comment:
What is this commit? It doesn't seem to exist? https://github.com/google/google-api-go-client/commit/48e49d1645e228d1c50c3d54fb476b2224477303